app/views/hearings/_hearing_events.html.haml

```diff
@@ -6,9 +6,10 @@
       %th.govuk-table__header{ scope: 'col' }= t('hearings.show.events.time')
       %th.govuk-table__header{ scope: 'col' }= t('hearings.show.events.event')
   %tbody.govuk-table__body
-    - hearing.hearing_events_for_day(hearing_day).each do |event|
-      %tr.govuk-table__row
-        %td.govuk-table__cell
-          = event.occurred_at.to_datetime.strftime('%H:%M')
-        %td.govuk-table__cell
-          = event.description
+    - if hearing.hearing_events.present?
```

Comment:
another potential solution, render the partial only if there are any hearing events:

`= render(partial: 'hearing_events', locals: { hearing: @hearing, hearing_day: @hearing_day }) if hearing.events`

on `app/views/hearings/show.html.haml` at line 9

Reply:
Thanks, I moved it over to the show view as that makes more sense